tests/unit/tests.inc

```diff
@@ -117,7 +119,9 @@ XX(update_from_header_no_sampled_flag)
 XX(update_from_header_null_ctx)
 XX(url_parsing_complete)
 XX(url_parsing_invalid)
+XX(url_parsing_no_path)
 XX(url_parsing_partial)
```

Comment:
I'm not sure where those are coming from.

Reply:
Joshua probably didn't run the test discovery before merging. It is not a biggie, but those files were not yet collected.